data/hai/unfettered 0 prologue.txt

```diff
@@ -137,6 +137,10 @@ mission "First Contact: Unfettered"
 		log "Factions" "Sheragi" `The Unfettered Hai say that a dragon-like species called the "Sheragi" who are long extinct used to live in human space, back when much of what is now human territory was occupied by the Hai.`
 
 
+outfit "Unfettered Landing License"
+	category "Licenses"
+	thumbnail "outfit/unknown"
+	description "This is the license that allows you to land freely on Unfettered planets so long as you act responsibly towards them."
```

Comment:
I think the licenses should be grouped together, likely in `hai outfits.txt`.

Reply:
good point but I think I'll delete it to make the ship itself be the license